core/services/aws/tests/user/cognitoTrigger/lambdas/postConfirmationTrigger.test.ts

```diff
@@ -0,0 +1,76 @@
+import postConfirmationLambda from '../../../../user/cognitoTrigger/lambdas/postConfirmationTrigger'
+import { PostConfirmationTriggerEvent } from 'aws-lambda'
+import { UserService } from '@application/userWorkflows/UserService'
+import DynamoDbTableOperations from '../../../../commons/ddb/DynamoDbTableOperations'
+import { UserDTO } from '@commons/dto/UserDTO'
+
+jest.mock('@application/userWorkflows/UserService')
+jest.mock('../../../../commons/ddb/DynamoDbTableOperations')
+
+describe('postConfirmationLambda Tests', () => {
+  let mockEvent: PostConfirmationTriggerEvent
+  let mockDynamoDbTableOperations: jest.Mocked<DynamoDbTableOperations<UserDTO, any, any>>
+
+  beforeEach(() => {
+    mockEvent = {
+      triggerSource: 'PostConfirmation_ConfirmSignUp',
+      request: {
+        userAttributes: {
+          email: 'ebrahim@example.com',
+          name: 'Ebrahim',
+          phone_number: '1234567890'
+        }
+      },
+      response: {},
+      region: 'us-east-1',
+      userPoolId: 'us-east-1_123456',
+      callerContext: {
+        awsSdkVersion: '1',
+        clientId: 'abc123'
+      },
+      version: '1',
+      userName: 'ebrahim@example.com'
+    } satisfies PostConfirmationTriggerEvent
+
+    mockDynamoDbTableOperations = {
```

Comment:
same as above

Reply:
Did the same for mockDynamoDBTableOperations too.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/6d856e5322c15066df61036318febceea7de3529)